config/locales/en.yml

```diff
@@ -11,6 +11,10 @@ en:
  click_here: "click here"
  less: " [Less]"
  language: 'Language'
+ true: 'yes'
```

Comment:
I was encountering some weird behaviour where when I specified yes and no to be the i18n keys, they got converted to true and false.

Reply:
Just saw found this: https://github.com/svenfuchs/i18n/issues/94
Could work if I did `'yes': 'yes'` for instance 🤔